build-tools/dependencies/dependencies.projitems

```diff
@@ -7,51 +7,61 @@
   <ItemGroup>
     <RequiredProgram Include="$(HostCcName)"        Condition=" '$(HostOS)' != 'Windows' " />
     <RequiredProgram Include="$(HostCxxName)"       Condition=" '$(HostOS)' != 'Windows' " />
-    <RequiredProgram Include="7za"                  Condition=" '$(NeedMxe)' == 'true' And $(AndroidSupportedHostJitAbisForConditionalChecks.Contains (':mxe-Win64:'))">
+    <RequiredProgram Include="x86_64-w64-mingw32-g++" Condition=" '$(HostOS)' == 'Darwin' " >
+      <Homebrew>mingw-w64</Homebrew>
+    </RequiredProgram>
+    <RequiredProgram Include="7za"                  Condition=" '$(HostOS)' == 'Darwin'">
       <Homebrew>p7zip</Homebrew>
     </RequiredProgram>
-    <RequiredProgram Include="autoconf"             Condition=" '$(HostOS)' != 'Windows' ">
+    <RequiredProgram Include="autoconf"             Condition=" '$(HostOS)' == 'Darwin' ">
       <Homebrew>autoconf</Homebrew>
     </RequiredProgram>
-    <RequiredProgram Include="automake"             Condition=" '$(HostOS)' != 'Windows' ">
+    <RequiredProgram Include="automake"             Condition=" '$(HostOS)' == 'Darwin' ">
       <Homebrew>automake</Homebrew>
     </RequiredProgram>
-    <RequiredProgram Include="cmake"                Condition=" '$(NeedMxe)' == 'true' And $(AndroidSupportedHostJitAbisForConditionalChecks.Contains (':mxe-Win64:'))">
+    <RequiredProgram Include="cmake"                Condition=" '$(HostOS)' == 'Darwin'">
       <Homebrew>cmake</Homebrew>
     </RequiredProgram>
-    <RequiredProgram Include="gdk-pixbuf-csource"   Condition=" '$(NeedMxe)' == 'true' And $(AndroidSupportedHostJitAbisForConditionalChecks.Contains (':mxe-Win64:'))">
+    <RequiredProgram Include="gdk-pixbuf-csource"   Condition=" '$(HostOS)' == 'Darwin'">
```

Comment:
I'm dubious of this and the other `Condition` changes.

Reply:
This is *probably* useful just for Mono and, again, Linux has a different set of dependency checks - therefore this is relevant only for macOS. `NeedMxe` was `true` only for macOS previously, so the condition effectively remains the same just shorter.